T.A.Colorify/index.css

```diff
@@ -0,0 +1,31 @@
+.color__div {
+	height: 25rem;
+	width: 25rem;
+	background-color: #10bcf4;
+	border-radius: 50%;
+	margin: 5% auto;
+}
+
+ button {
+	padding: 15px 32px;
+    text-align: center;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+}
```

Comment:
Deploy it on netlist and add the link in the readme.

Reply:
https://zealous-neumann-62dd8d.netlify.app